### ISSUE TITLE
Fix build on Node 12+

### ISSUE THREAD

**Issue by mattcollier** [digitalbazaar/rdf-canonize-native#9]
This module is no longer building on node 12+

See: https://travis-ci.org/github/digitalbazaar/rdf-canonize/builds/731728025

**Comment by dmfilipenko-affinidi** [comment]
Hey @mattcollier, thanks for awesome work. Any plans to fix this issue? Do you need any help with this?

**Comment by mattcollier** [comment]
@dmfilipenko-affinidi Hello, this module was created at a time when this native implementation was more performant than JS.  This module has fallen into disuse since the JS/V8 implementation caught up performance wise.

**Comment by danbri** [comment]
Maybe a WASM build would fill whatever gap is left?

**Comment by davidlehn** [comment]
@danbri If someone is interesting in tackling this, I'd suggest first building the native code on an older Node.js, and benchmarking that against the pure js version on a modern Node.js.  I have no idea what the result will be.  I'd put my money on about even or a js edge. ;-)

Someone with more experience with the modern native API could probably update this code fairly quickly too.

I'd also suggest revisiting benchmarks and profiling on the pure js code before going to far down the native code route.  There might be algorithmic gains there for less effort.  (There's at least one outstanding patch for a small optimization.)

**Comment by danielbakas** [comment]
Hi all! Hope you are doing great.

I encountered an issue (https://github.com/digitalbazaar/rdf-canonize-native/issues/10) that might be related to this.

Is this issue still present?

Thanks in advance!

**Comment by davidlehn** [comment]
Yes this issue is still not addressed and likely won't be anytime soon by us.  (PRs welcome!)  See:
https://github.com/digitalbazaar/jsonld.js/issues/567#issuecomment-2707527873
